src/components/OperationList.tsx

```diff
@@ -0,0 +1,171 @@
+import * as React from "react";
+import { connect, MapStateToProps } from "react-redux";
+
+import styled from "styled-components";
+import { green50 } from "material-ui/styles/colors";
+
+import { Uid } from "../identifiers";
+import { AppState } from "../store";
+import { Member } from "../reducers/membersNew";
+import { Operation, OperationType } from "../reducers/operations";
+import { getMembersByUid } from "../selectors/members";
+
+import IntlMessage from "./IntlMessage";
+
+// styling from: https://material-components.github.io/material-components-web-catalog/#/component/list
+const OperationList = styled.ul`
+  padding: 8px 0px;
+  background-color: ${green50};
+  border: 1px solid rgba(0, 0, 0, 0.1);
+
+  > li {
+    list-style-type: none;
+    height: 48px;
+    padding: 0px 16px;
+    display: flex;
+    align-items: center;
```

Comment:
Why does this need to be flex if it only ever has one element in it? Needless complexity in style

Reply:
For give operations it also includes secondary text.